js/app.js

```diff
@@ -1,19 +1,63 @@
 /*-------------------------------- Constants --------------------------------*/
+const characters = ['❤️', '💰', '⚽', '💎', '🤑']
+const winningMessage =  `You have won`
+const winVoice = new Audio('Win.mp3')
+/*-------------------------------- variables--------------------------------*/
+ let char 
+ let wins = 0
 
+/*-------------------------------- Cached Element References--------------------------------*/
 
-
-/*-------------------------------- Variables --------------------------------*/
-let youWin, 
-
-
-
-/*-------------------------------- Constants Element References--------------------------------*/
-const machine
-
+var button = document.querySelector('.spin')
+var  win = document.getElementById('win')
+var winnings = document.getElementById('score')
 
 /*----------------------------- Event Listeners -----------------------------*/
-
-
-
-
-/*-------------------------------- Functions --------------------------------*/
\ No newline at end of file
+button.addEventListener('click', function(event){
+  return spin()
+})
```

Comment:
You don't need to return `spin` here! You can just call the `spin` function when the spin button is clicked with:

```javascript
button.addEventListener('click', spin)
```